Makefile.NSE.Win

```diff
@@ -3,7 +3,8 @@
 VPROC_GMAKE=$(shell sh ./version.sh MAIN)
 
 # Set up environment to use product compiler.
-export COMP_ROOT=C:\Program Files (x86)\HPE NonStop\J06.20
+NONSTOPOSVERSION?=J06.20
```

Comment:
This is just the default if otherwise unset, right?

Reply:
Exactly, So passing `NONSTOPOSVERSION=J06.23` on the make line would change COMP_ROOT.